modules/DesktopProject/src/main/java/org/gephi/desktop/project/ProjectControllerUIImpl.java

```diff
@@ -281,8 +284,9 @@ public void approveSelection() {
             NbPreferences.forModule(ProjectControllerUIImpl.class).put(LAST_PATH, file.getAbsolutePath());
 
             // Save file
-            saveProject(controller.getCurrentProject(), file);
+            return saveProject(controller.getCurrentProject(), file);
         }
+        return null;

```

Comment:
Probably better to return `CompletableFuture.completedFuture()`

Reply:
Done